server/db/006-addheartbeat.sql

```diff
@@ -0,0 +1,31 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 6;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        CREATE TABLE IF NOT EXISTS heartbeat (
```

Comment:
Similar to what Dave said about the `Heartbeat` class, I think that `heartbeat` isn't the greatest table name. Maybe `hubs` like he suggested. Whatever it ends up being, would be good if it were plural to match `installations`, `sessions`, and `migrations`.